src/test/java/com/sendgrid/SendGridTest.java

```diff
@@ -35,6 +32,15 @@ public void testInitialization() {
   }
 
   @Test
+  public void testConstructWithClient() throws IOException {
+    Client client = mock(Client.class);
+    SendGrid sg = new SendGrid(SENDGRID_API_KEY, client);
+    Request request = new Request();
+    sg.makeCall(request);
+    verify(client).api(request);
```

Comment:
instead of using a mock I could also add a getter for the client and make sure its the one we passed into the constructor. Opinions? :blush:

Reply:
I am sending Email with attachments. I want to set ConnectionTimeout values like shown above, But Not able to figure out where to set. I created Client object. I used abovbe 4 lines. But I do no see any API on Client or SendGrid to set timeout values. Can someone please help me with this.